Code/Framework/AzToolsFramework/AzToolsFramework/Archive/ArchiveComponent.cpp

```diff
@@ -158,16 +158,16 @@ namespace AzToolsFramework
 
             bool success = true;
             AZStd::vector<char> fileBuffer;
-            const AZ::IO::Path workingPath{ dirToArchive };
+            const AZ::IO::FixedMaxPath workingPath{ dirToArchive };
 
             for (const auto& fileName : foundFiles.GetValue())
             {
                 bool thisSuccess = false;
 
-                AZ::IO::PathView relativePath = AZ::IO::PathView{ fileName }.LexicallyRelative(workingPath);
+                AZ::IO::FixedMaxPath relativePath = AZ::IO::FixedMaxPath{ fileName }.LexicallyRelative(workingPath);
```

Comment:
PathView is a `string_view` backed path type.
The `PathView::LexicallyRelative` function returns an AZ::IO::FixedMaxPath which is an `AZStd::fixed_string` backed type.

Alternatively `auto` would work as well.